util/src/test/java/io/kubernetes/client/PortForwardTest.java

```diff
@@ -110,6 +111,12 @@ public void testPortForwardResult() throws IOException, InterruptedException {
         makeStream(new byte[] {2, 32, 3}, msgData2.getBytes(StandardCharsets.UTF_8)));
 
     final Object block = new Object();
+    final Semaphore s= new Semaphore(1);
+    try{
```

Comment:
I don't think you need the try/catch block. This function can throw InterruptedException already (though it won't in practice)